registration-processor/pre-processor/registration-processor-operator-validator-stage/src/main/java/io/mosip/registration/processor/stages/operatorvalidator/OperatorValidator.java

```diff
@@ -362,9 +362,9 @@ private boolean validateMapping(List<String> pathsegments, InternalRegistrationS
 		regProcLogger.debug("validateMapping call ended for registrationId {} with response data {}",
 				registrationStatusDto.getRegistrationId(), JsonUtil.objectMapperObjectToJson(userDto));
 		if (userDto != null) {
-			if (responseWrapper.getErrors() == null) {
-				isValidUser = userDto.getRegistrationCenters().get(0).getIsActive();
-			} else {
+			isValidUser = userDto.getRegistrationCenters().get(0).getIsActive();
+		} else {
+			if (responseWrapper.getErrors() != null) {
```

Comment:
can we rather first check if errors are there, if yes return mapping not found, in else if check userDTO != null and in final else log a waring that error is not there and userDTO is empty and return false in that case ?

Replicate the same for supervisor also

Reply:
reconstructed this if else condition...